src/pyobo/registries/metaregistry.json

```diff
@@ -1262,4 +1263,4 @@
   "wikidata_property_types": [
     "Q42396390"
   ]
-}
```

Comment:
leave the missing trailing new line. the JSON exporter from python doesn't put one and I don't want to have this on the diff every time